### DISCUSSION TITLE
step ca init --pki --name="Tiny"

### DISCUSSION THREAD

**Discussion by geekmarine** [smallstep/certificates#484] [Q&A] [answered]
I'm having issues getting past this step. Can anyone help?

ubuntu@caserver:~$ sudo step ca init --pki --name="Tiny"
MoTTY X11 proxy: Unsupported authorisation protocol
qt.qpa.xcb: could not connect to display localhost:10.0
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the applicati                                               on may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayl                                               and-xcomposite-egl, wayland-xcomposite-glx, xcb.

Aborted

**Comment by maraino** [comment, marked as answer]
After googling this error, it looks like your user doesn't have the authorization to connect to the X server, we don't use the UI so my guess is that sudo is trying to display the password in the X server.

Here's some things that you can try:
* If you want to run it with the user root:
  * `sudo -E step ca init...`
  * `sudo su` and then `step ca init ...`
* Use your normal user, and then move the `~/.step` directory to the desired location and change the owner with `chown`

See a full explanation and solutions here:
https://kerneltalks.com/troubleshooting/mobaxterm-x11-proxy-authorisation-not-recognised/
https://superuser.com/questions/912669/error-putty-x11-proxy-authorisation-not-recognised
https://chadrick-kwag.net/x11-proxy-unsupported-authorisation-protocol-error-fix/

**Reply by geekmarine** [reply]
Thanks for the help. I'll give this a shot this weekend.

Much appreciation for this tutorial. I'm using your tutorial for learning purposes.
